packages/lesswrong/lib/collections/users/views.ts

```diff
@@ -110,7 +110,7 @@ Users.addView('LWUsersAdmin', (terms: UsersViewTerms) => ({
 Users.addView("usersWithBannedUsers", function () {
   return {
     selector: {
-      bannedUserIds: {$exists: true}
+      $or: [{bannedPersonalUserIds: {$ne:null}}, {bannedUserIds: {$ne:null}}]
```

Comment:
Did you know that this would have been a bug until very recently?

Reply:
_Very_ recently: https://github.com/ForumMagnum/ForumMagnum/pull/6618/files#diff-dff188afe3a43c2ed4ddd6a3daa618395937c99ab910204e89a70e15c0473a3b

Reply:
Or maybe not, depends on if the default view of users has an `$or`, which, do I remember that there isn't one?

Reply:
I... did not know that. (I did start by filtering for "exists: true" which explicitly didn't work for bannedPersonalUserIds for some reason)